lldb/source/Plugins/ExpressionParser/Clang/ClangExpressionParser.cpp

```diff
@@ -578,7 +657,12 @@ static void SetupImportStdModuleLangOpts(CompilerInstance &compiler) {
   lang_opts.GNUMode = true;
   lang_opts.GNUKeywords = true;
   lang_opts.CPlusPlus11 = true;
-  lang_opts.BuiltinHeadersInSystemModules = true;
+
+  // FIXME: We should use the driver to derive this for us.
+  // ClangModulesDeclVendor already parses the SDKSettings for the purposes of
+  // this check.
```

Comment:
we should put another FIXME here (we can fix that later) that we should use DW_AT_LLVM_sdk and then look up a matching local SDK using HostInfo, instead of the include dirs.